lldb/tools/lldb-dap/JSONUtils.cpp

```diff
@@ -1103,28 +1116,52 @@ std::string CreateUniqueVariableNameForDisplay(lldb::SBValue v,
 //                       can use this optional information to present the
 //                       children in a paged UI and fetch them in chunks."
 //     }
+//
+//
+//     // Unofficial extensions to the protocol:
+//
```

Comment:
Do we want to add these inside of an object with a unique name that VS code will never use? Like:
```
{ "$__lldb_extensions": { 
    "inMemoryValue": true, 
    "summary": "...", 
    "autoSummary": "..." 
  }
}
```

Reply:
Good call!